llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1287,7 +1289,9 @@ bool SPIRVToLLVM::postProcessRowMajorMatrix() {
 
             for (unsigned i = 0; i < cast<FixedVectorType>(pointerType)->getNumElements(); i++) {
               Value *const pointerElem = getBuilder()->CreateExtractElement(pointer, i);
-              Type *const newLoadElemType = pointerElem->getType()->getPointerElementType();
+              Type *const newLoadElemType = pointerElemType;
+              // TODO: Remove this when LLPC will switch fully to opaque pointers.
+              assert(IS_OPAQUE_OR_POINTEE_TYPE_MATCHES(pointerElem->getType(), newLoadElemType));
```

Comment:
newLoadElemType is always equal to pointerElemType, right? If so, can you just remove that variable?

Reply:
Thanks ! pushed update.